### PR TITLE
Prep for release

### PR DESCRIPTION
**Pull request by JayGreeeen**
### :link: Related Issues

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## [16.2.0] - 2020-06-30
+## [16.2.0] - 2020-07-01
```
Comment (abhimanyugupta07) [inline CHANGELOG.md:1]: So, you are releasing 16.2.0 but snapshot is 16.1.1 ?

File: circus-train-api/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>circus-train-parent</artifactId>
-    <version>16.2.1-SNAPSHOT</version>
+    <version>16.1.1-SNAPSHOT</version>
```
Comment (massdosage) [inline circus-train-api/pom.xml:7]: Shouldn't this be 16.2.0-SNAPSHOT? (although you can override it in Jenkins when you do the release so doesn't matter too much)

Comment (JayGreeeen) [reply circus-train-api/pom.xml:7]: I just reverted the changes that the maven release made and put it back to what it was before, so when I release I'll add the version then.

File: CHANGELOG.md (hunk 1 above)
Comment (JayGreeeen) [reply CHANGELOG.md:1]: Yeah that will be changed when I perform the release

**Pull request merged**